content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -0,0 +1,261 @@
+information.
+
+When the customer name is added, the :guilabel:`Sales Order Item` field automatically populates with
+the most recent prepaid sales order item that has time remaining.
+
+If a customer has more than one sales order item with remaining time, click the :guilabel:`Sales
+Order Item` field and select the correct item from the drop-down list.
+
+After entering all of the necessary information, click :guilabel:`Save`.
+
+Track hours on helpdesk ticket
+------------------------------
+
+Time spent working on a *Helpdesk* ticket is tracked on the *Timesheets* tab on the specific ticket.
+
+On the ticket detail form, click on the :guilabel:`Timesheets` tab and click :guilabel:`Add a line`.
+Choose an :guilabel:`Employee`, add a :guilabel:`Description` of the task, and enter the number of
+:guilabel:`Hours Spent` working on the task.
+
+As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field,
+at the bottom-right of the tab, is automatically updated.
+
+.. image:: track_and_bill/track-bill-remaining-hours-total.png
+   :align: center
+   :alt: The timesheets tab of a Helpdesk ticket keeping track of the number of hours remaining on a
+       sales order.
+
+.. note::
+   If the number of hours on the :guilabel:`Timesheets` tab exceeds the number of hours sold, the
+   :guilabel:`Remaining Hours of SO` turns red.
+
+As hours are added to the :guilabel:`Timesheets` tab, they are automatically updated in the
+:guilabel:`Delivered` field on the :abbr:`SO (sales order)`, as well.
+
+Invoice post-paid support services
+==================================
+
+When support services are billed based on the amount of time spent on an issue, an invoice
+**cannot** be created before the total number of hours required to solve the problem have been
+entered on a timesheet. In this case, a service product with the *Invoicing Policy* set to *Based on
+Timesheets* would be used, like the one created :ref:`above <helpdesk/configure-service-products>`.
+
+Create a sales order with a time-tracked product
+------------------------------------------------
+
+To invoice a customer for post-paid support services, first create a sales order (SO) with the
+:guilabel:`Service on Timesheets` product. To do this, go to :menuselection:`Sales app --> Orders
+--> Quotations`. Then, click :guilabel:`Create` to reveal a blank quotation form.
+
+Fill out the quotation with the customer information.
+
+On the :guilabel:`Order Lines` tab, click :guilabel:`Add a Product`. Select the :guilabel:`Service
+on Timesheets` product configured :ref:`above <helpdesk/configure-service-products>`. After updating
+any other necessary information, :guilabel:`Confirm` the quotation.
+
+.. important::
+   Unlike with the prepaid service quotations, *Odoo* does **not** allow an invoice to be created at
+   this time. That is because no services have been performed; in other words, nothing has been
+   delivered, therefore, there is nothing to invoice.
```

Comment:
I am actually still seeing a 'Create Invoice' button after confirming a quotation with just the 'Service on Timesheet' product in the Order Lines tab. Your note makes *total* sense, so it might be worth a quick double-check about this admonition during your revisions.